app/javascript/app/components/vulnerability-graph/vulnerability-graph-component.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import CircularChart from 'components/circular-chart';
+import InfoIcon from 'components/info-icon';
+import { COUNTRIES_NUMBER } from 'data/country-codes';
```

Comment:
I'd prefer to calculate this number dynamically with the number of countries in the request.
Is that possible? Maybe in the action ?